docs/install/containers/images/backend.md

```diff
@@ -65,6 +65,13 @@ Plone 6 example:
 docker run -p 8081:8081 -e LISTEN_PORT=8081 plone/plone-backend:{PLONE_BACKEND_VERSION}
 ```
 
+#### Adding configuration to `zope.conf` or additional ZCML
+
+With the standard container, it is not possible to add configuration fragments to
+`zope.conf` directly or add extra ZCML, like it is with the `buildout` deployment
+method.  However, you can derive your own container image, and drop in configuration
+fragments.  See *Extending from this image* below for instructions.
```

Comment:
Change this to a `ref`.

```suggestion
fragments.  See {ref}`backend-extending-from-this-image-label` below for instructions.
```

Also above the section heading below, insert a label.

```
(backend-extending-from-this-image-label)=

## Extending from this image
```

Reply:
It's done.

Reply:
I don't see the label inserted on line 288. Did you push the commit?

Reply:
I merged it directly via Github.  WTF.

Reply:
No, that line you had do separately. I was not able to insert a suggestion in something you had not edited. /me shakes fist at GitHub UI not doing everything under the sun.